EIPS/eip-7517.md

```diff
@@ -0,0 +1,607 @@
+---
+eip: 7517
```

Comment:
```suggestion
eip: 7518
```
Assigning next sequential EIP/ERC number.  7517 already issued to #7682 

Please also update the filename.